example/01-dwd-prober-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
```

Comment:
Please rename the files in `example` directory to:
```
01-dwd-prober-configmap.yaml
02-dwd-weeder-configmap.yaml
03-dwd-prober-deployment.yaml
04-dwd-weeder-deployment.yaml
```
This maintains the correct order of components to be deployed.

Reply:
Changed.